### DISCUSSION TITLE
isotropic material between two layups

### DISCUSSION THREAD

**Discussion by sange000000** [wenbinyugroup/.github#32] [VABS Related Discussions]
![image](https://github.com/user-attachments/assets/2026f8fd-790f-4a8e-a2f5-36654c6c87ed)
The attached image is a isotropic material inside a box layup, instead I wish to have an iso fill between two layups at top and bottom(without the sides). Could please tell how this can be done?

**Comment by sange000000** [comment]
I would like to do this without defining the lamina/layup (ie not mentioning thickness). Using a fill option, with baseline and top boundary

**Reply by wenbinyu-PU** [reply]
How about have the two sides of the box made of the same material as the filling material. Then you essentially trick the code to achieve what you want.

**Reply by sange000000** [reply]
Thanks, that trick works. But I find that any component after the first fill component is not read, example here

![image](https://github.com/user-attachments/assets/2c6e0baf-035e-4e29-8276-8a5470592fcc)
The last component 'bondfill2' is ignored. I can place the fill component to be the last component. But I want to have 2 iso fills in my section, What to do here?

**Reply by unitasium** [reply]
There seems an extra `</segment>` tag in the component `bondFill1`. Can you delete this and try it again?
